outplacement_final_report/models/outplacement.py

```diff
@@ -32,31 +34,26 @@ class OutplacementGoal(models.Model):
     # return codes for these two selections should be limited to search
     # ilike field_of_work.code
     job_id = fields.Many2one(comodel_name="res.ssyk", string="Job title")
-    job_description = fields.Char(string="Job description")
-    motivation = fields.Selection(selection=[
-        ('matches interest', 'Matchar deltagarens intressen'),
-        ('matches ability', 'Arbetsuppgifter matchar förmåga'),
-        ('market demand', 'Efterfrågan på arbetsmarknaden'),
-        ('complementing education', 'Kompletterar nuvarande utbildning'),
-        ('complementing experience', 'kompletterar tidigare erfarenheter'),
-        ('other', 'Other')
-        ], string="Motivation")
-    # should only be used if motivation is "other"
-    free_text = fields.Char(string="Free text")
     step_ids = fields.Many2many(
         comodel_name="outplacement.goal.step", string="Step")
+    job_description = fields.Char(string="Job description") 
```

Comment:
Trailing comas in field definitions makes Odoo sad

Reply:
I don't see it/get it